README.md

```diff
@@ -24,9 +24,26 @@ We currently support the following functional forms. Parameters that can be opti
 4. PBC LennardJones 612 (sigma, epsilon)
 5. PBC reaction field electrostatics (charge)
 
-## Running Tests
+## Installation
 
-Note: we currently only support and test on python 3.7, use other versions at your own peril.
+### Pre-requisites
+
+* RDKit
+* OpenEye Toolkits
+* OpenMM
+* Cuda 11.0+
```

Comment:
I am guessing this supports Cuda 10+, just not familiar enough with the code yet to say and I tried this with Cuda 11.0

Reply:
I'm testing whether or not we should switch to CUDA 11 right now, it would be nice to not have to configure cub in the CMakeLists.txt file.